pages/dashboard/events/[id].tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+
+const Event: React.FC = () => {
+  const router = useRouter();
+  const { id } = router.query;
+
+  return <h1>You&apos;re on event: {id}</h1>;
```

Comment:
You can just use an apostraphe in `'` :P

Reply:
Me: '
Eslint: 😡
hahahahah